packages/js/eslint-plugin/src/configs/index.ts

```diff
@@ -102,6 +102,7 @@ export const project: TSESLint.Linter.ConfigType = {
               "expectEventPayloadToMatch",
               // Shared assertion for visual regression tests
               "expectSnapshot",
+              "expectScreenshotAreaSnapshot",
```

Comment:
I couldn't find this used anywhere. Is it left over from a previous version, and do we still need to include it?

Reply:
This is used in storybook visual regression tests, e.g. `frontend/test/storybook/visual-regression/v-checkbox.spec.ts`. It makes testing the screenshot area (instead of the full page) easier.

Reply:
Ah! A diff must have been collapsed when I "ctrl + f" searched for it. Sorry!